### DISCUSSION TITLE
Validation of an object in use-form state

### DISCUSSION THREAD

**Discussion by Uzaaft** [mantinedev/mantine#1542] [Q&A] [answered]
I currently have a state which has an object, similar to this: 
```typescript
const form = useForm({
        initialValues: {
            fractions: {},
        },
    })
```

The fractions object will be having the following data:
```typescript
{fractionName: Number}
```
So that the object can for.ex be:
```typescirpt
{"Tim": 10, "Kyle": 50}
```
Etc...
I would like to validate the numbers, to make sure that they are not higher than 100, but I am struggling trying to implement this. Anyone here who could help me with this? Or tell me if I am over-thinking my implementation?

**Comment by rtivital** [comment, marked as answer]
use-form does not support nested objects

**Reply by Uzaaft** [reply]
Thank you for the quick response! If I can ask, how would you do this with Mantine?

**Reply by rtivital** [reply]
I use only plain objects in my forms
